README.md

```diff
@@ -1,45 +1,37 @@
 # Satellite-clone
+
 Satellite-clone contains simple Ansible playbooks that can be used to perform any of the following tasks:
-* Setup Satellite 6.1 or 6.2 install with the Satellite backup data.
+* Setup a Satellite 6.1 or 6.2 install with restored backup data.
 * Change the hostname on an existing Satellite install.
-* Perform a minor upgrade of an existing Satellite.
+* Perform a minor upgrade of an existing Satellite. 
 
 ## Getting Started
-Ideally, you need two hosts to run this project:
-
-1. Ansible Control node (referred to as `Control node` in the rest of this document) is the host from which this ansible project is run.
-2. Destination node - This must be one of the following:
-    - A blank machine - to clone an existing Satellite Server.
-    - A Satellite server - for a hostname change or for Updating Satellite.
 
-**Note**
+#### You will need: ####
 
-1. You can get away with using one host by optionally choosing to use the `Destination node` as the `Control node`.
-2. Make sure that the `Control node` can connect to the `Destination node` via paswordless ssh.
+**to clone a Satellite Server**:
+    - A blank (vanilla install) RHEL 7 server. You will run the setup commands here.
```

Comment:
I don't remember the exact discussions around supporting just rhel7.  But I thought we wanted to support this scenario:  a user cloning a satellite6 machine from a rhel6 machine to another rhel6 machine, perform upgrade of satellite to see if everything works, then change hostname and make the new cloned box as the main satellite server.  He can then choose to migrate to rhel7 whenever needed.